index.html

```diff
@@ -156,23 +156,23 @@ <h3 id="oneweb" data-export="" data-dfn-type="dfn">
 
 <section>
   <h3 id="noharm" data-export="" data-dfn-type="dfn">
-The web does not cause harm to society
+An ethical web avoids harms to people
   </h3>
-  <p>
-When we are adding a feature or technology to the web,
-we will work to prevent or mitigate any harm it might cause society or groups,
-especially to vulnerable people.
```

Comment:
losing this phrase is not great.

Reply:
Is this just the "especially vulnerable people" or does the following do this?

> In adding technology or features to the web,
> we commit to first understanding how all parties might be affected.
> Where changes have harmful consequences,
> we will either prevent or mitigate those harms to the extent possible.

I can add ", especially those who are most vulnerable or disadvantaged" to the first, if it is just that one clause.

Reply:
that would work for me!